CHANGELOG.md

```diff
@@ -54,18 +54,18 @@ sound_out FX
 
 * New synth `live_audio` - directly stream audio from your soundcard as a synth.
 
-* New FX `record` - enables you internally record any audio into named buffers. Perfect for building looper systems.
+* New FX `record` - enables you to internally record any audio into named buffers. Perfect for building looper systems.
 
-* New FX `:sound_out` - stream out a audio to a specific output on your sound card. This enables multi-channel audio out. 
-* New FX `:sound_out_stereo` - similar to `sound_out` but streams out to pair of consecutive audio card (left and right) output channels.
+* New FX `:sound_out` - stream out audio to a specific output on your sound card. This enables multi-channel audio out.
+* New FX `:sound_out_stereo` - similar to `sound_out` but streams out to a pair of consecutive audio card (left and right) output channels.
 * New FX `eq` - Parametric EQ with three centre freqs - low, mid & high - all with Q values and gain (-1 -> 1). Also has low and high shelves with centre freqs and slope adjustment.
-* New FX `tremelo` - simple tremelo effect which modulates the volume within the `do/end` block.
+* New FX `tremolo` - simple tremolo effect which modulates the volume within the `do/end` block.
```

Comment:
Ah, I get you - actually luckily the FX is correctly named - so it's just a typo in the changelog :-)